src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+body {
```

Comment:
You shouldn't use styles directly on `body` tag in your block scss file. It will be better to move these styles to global scss file or remove them if they are not essential.